packages/alfa-aria/src/features/tr.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  Element,
+  getClosest,
+  isElement,
+  getTagName,
+  Node
+} from "@siteimprove/alfa-dom";
+import { getRole } from "../get-role";
+import { Feature, Role, AnyRole } from "../types";
+import * as Roles from "../roles";
+
+/**
+ * @see https://www.w3.org/TR/html-aria/#tr
+ */
+export const Tr: Feature = {
+  element: "tr",
+  role,
+  allowedRoles: AnyRole
+};
+
+function role(tr: Element, context: Node): Role | undefined {
+  if (
+    getClosest(tr, context, tr => {
+      return (
+        isElement(tr) &&
+        getTagName(tr, context) === "table" &&
+        getRole(tr, context) === Roles.Grid
+      );
+    }) !== null
+  ) {
```

Comment:
The `<tr>` element always has an implicit role of `row`, though this role _can_ be made explicit if the parent `<table>` has a role of `grid`. `role: Roles.Row` is therefore sufficient.